app/views/instruments/show.html.erb

```diff
@@ -3,6 +3,12 @@
 
 <h1><%= @instrument.name %></h1>
 <p><%= @instrument.description %></p>
-<p>Capacity: <%= @instrument.capacity %></p>
 <p>Price: <%= @instrument.price %></p>
 <%= link_to 'Back to Instruments', instruments_path %>
+
+
+<h2>Book</h2>
```

Comment:
I think we need to have a gate here to only allow logged in users to book. I thought easiest would thus be a booking new page that can only be reached by users. So there could be a link here and the form on the bookings new page.

Reply:
Yes, that's how I have it on this branch, after seeing your user flow on Figma. With "before_action :authenticate_user!" on the application controller and a "skip_before_action :authenticate_user!, only: [:index, :show]" on the instruments controller. If you click on "book" you are prompted to login before you see the booking new page.